### DISCUSSION TITLE
More zoom-in?

### DISCUSSION THREAD

**Discussion by MadeInDutch** [OpenRCT2/OpenRCT2#17039] [Feature requests]
At the moment, OpenRCT2 zooms in about this much max...

![image](https://user-images.githubusercontent.com/67276601/164896078-657bdfc8-1ac9-4066-8ec4-1196247cefe9.png)

...while RollerCoaster Tycoon Classic zooms in about this much. (I had to use the in-built screenshot feature as "print screen" doesn't work)

![screenshot_20220423_15_21_23](https://user-images.githubusercontent.com/67276601/164896515-1383def4-b4ce-4f0e-ae9f-a1e4245946dc.png)

It might seem minor but I do really want to zoom into my park, maybe to check for vandalism, or just for my own enjoyment. I feel it's the one thing RCT Classic inarguably does better than OpenRCT2. Surely it shouldn't be impossible to implement more zoom-in?

**Comment by IntelOrca** [comment]
@MadeInDutch if you switch to OpenGL, you can zoom in further.

**Reply by MadeInDutch** [reply]
oh ok that works

thanks
